### PR TITLE
Use Tag-Helpers

### PR DESCRIPTION
**Pull request by TravisBoatman**
Addresses #1284

### CODE REVIEW

**Comment by rachelappel** [comment]
@Rick-Anderson review

File: aspnet/mvc/controllers/areas.rst [hunk 1]
```diff
@@ -89,19 +89,19 @@ Now, when the user browses to *http://<yourApp>/products*, the ``Index`` action
 Linking between areas
 ---------------------
 
-To link between areas, you simply specify the area in which the controller is defined. If the controller is not a part of an area, use an empty string.
+To link between areas, you simply specify the area in which the controller is defined by using `Tag-Helpers <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Mvc/TagHelpers/ImageTagHelper/index.html>`__.
```
Comment (danroth27) [inline aspnet/mvc/controllers/areas.rst]: > using `Tag-Helpers <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Mvc/TagHelpers/ImageTagHelper/index.html>`__

using :doc:\`Tag Helpers \</mvc/views/tag-helpers/index\>\`.

**Comment by danroth27** [comment]
@TravisBoatman One minor change

Otherwise :shipit:

**Comment by rachelappel** [comment]
Excellent work, thanks Travis! 53f54c7ecb2b16568dcb159374d9e1d0d587a227

**Pull request closed without merging**